pandas/core/generic.py

```diff
@@ -285,22 +285,6 @@ def _init_mgr(
                 mgr = mgr.astype(dtype=dtype)
         return mgr
 
-    @classmethod
-    def _from_mgr(cls, mgr: Manager):
```

Comment:
Removed this, because it was only used in the groupby Splitter (and so that usage was removed in the previous PR)